AudioKit/Common/Internals/AKSettings.swift

```diff
@@ -74,6 +74,22 @@ open class AKSettings: NSObject {
         }
     }
 
+    #if !os(macOS)
+    /// Whether haptics and system sounds are muted while a microhpone is setup or recording is active
+    @objc public static var allowHapticsAndSystemSoundsDuringRecording: Bool = false {
+        didSet {
+            if #available(iOS 13.0, *) {
+                do {
+                    print("allow them haptics")
```

Comment:
@warpling Can you make this print an "AKLog" and improve the grammar. :) Or you can take the line out. Or if you want I can accept the pr and remove it myself.

Reply:
haaaaaa totally accident. Can't believe I missed that.